gammapy/data/pointing.py

```diff
@@ -441,26 +452,27 @@ def get_altaz(self, obstime=None, location=None) -> SkyCoord:
     @property
     @deprecated("1.1")
     def meta(self):
+        """Meta header info as a dictionary."""
         if self._meta is not None:
             return self._meta
         return dict(self.to_fits_header(time_ref=self._time_ref))
 
     @property
     @deprecated("1.1")
     def location(self):
-        """Observatory location (`~astropy.coordinates.EarthLocation`)."""
+        """Observatory location as an `~astropy.coordinates.EarthLocation` object."""
         return self._location
 
     @property
     @deprecated("1.1")
     def time_start(self):
-        """Start time (`~astropy.time.Time`)."""
+        """Start time as a `~astropy.time.Time` object."""
```

Comment:
```suggestion
        """Start time as an `~astropy.time.Time` object."""
```

Reply:
It should be "a" here because `~astropy.time.Time` will only display Time in the doc.